README.md

```diff
@@ -51,6 +61,19 @@ The links for each required repo are listed below,
 - [Concorda][]
 - [Vidi][]
 
+To clone from a users repo the command takes the following structure:
```

Comment:
64 > If you have git available at the command line you can clone each repo using commands in the form,